### PR TITLE
[RFR]Check none value for key pair

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Check none value for key pair in provision form
{{pytest: -vvv -k test_select_key_pair_none_while_provisioning}}

https://polarion.engineering.redhat.com/polarion/#/project/RHCF3/workitem?id=RHCF3-12722

### CODE REVIEW

File: cfme/tests/cloud/test_providers.py [hunk 1]
```diff
@@ -303,6 +305,37 @@ def test_openstack_provider_has_api_version():
     assert view.api_version.is_displayed, "API version select is not visible"
 
 
+@pytest.mark.tier(3)
+@pytest.mark.usefixtures('has_no_cloud_providers')
```
Comment (quarckster) [inline cfme/tests/cloud/test_providers.py]: **QUESTION**
Why is `has_no_cloud_providers` not in the test arguments?
```python
def test_select_key_pair_none_while_provisioning(request, has_no_cloud_providers):
```

Comment (rlbabyuk) [reply cfme/tests/cloud/test_providers.py]: hm, not a good reason here, yup better to move it

File: cfme/tests/cloud/test_providers.py [hunk 2]
```diff
@@ -303,6 +305,36 @@ def test_openstack_provider_has_api_version():
...
+        Requirement: Have an ec2 provider with single key pair
+                    (For now available in South America (Sao Paulo) region)
+        1. Compute -> Cloud -> Instances
+        2. Click on Provision Instances in Toolbar
+        3. Go to Properties
+        4. Select None in Guest Access Key Pair
+        5. None should be selected
+    """
+
+    prov_config = conf.cfme_data['management_systems']['ec2_sa-east-1']
```
Comment (mfalesni) [inline cfme/tests/cloud/test_providers.py]: I don't like seeing hardcoded keys like `ec2_sa-east-1` here, can this be solved by our provider parametrizer?

Comment (rlbabyuk) [reply cfme/tests/cloud/test_providers.py]: hm, not sure here, as this region are not commonly used in our tests

Comment (rlbabyuk) [reply cfme/tests/cloud/test_providers.py]: Updated provider use, will update only region that needed for a test

**Comment by dajoRH** [comment]
Lint report for commit c765b931428ab649bb9a622749da02368823603a:

`cfme/tests/cloud/test_providers.py`:
- :red_circle: Line 23:1: **F401** *'cfme.utils.conf' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**